style.css

```diff
@@ -0,0 +1,23 @@
+* {
+    box-sizing:border-box;
```

Comment:
if you added reset.css, you don't need to add this style, because it is already used in reset.css